openy.info.yml

```diff
@@ -154,6 +154,7 @@ dependencies:
   - openy_prgf_blog_listing
   - openy_prgf_block_date
   - openy_prgf_loc_finder
+  - openy_prgf_location_by_amenities
```

Comment:
Extend openy.install with hook_update_N and enable this module from there, please @kolesnikoff